src/Verilator.cpp

```diff
@@ -617,6 +577,103 @@ int main(int argc, char** argv, char** env) {
+        cmd << v3Global.opt.getenvMAKE();
+        cmd << " -C " << v3Global.opt.makeDir();
+        cmd << " -f " << v3Global.opt.prefix() << ".mk";
+        if (jobs == 0) {
+            cmd << " -j";
+        } else if (jobs > 1) {
+            cmd << " -j " << jobs;
+        }
+        for (V3StringList::const_iterator it = makeFlags.begin(); it != makeFlags.end(); ++it) {
+            cmd << ' ' << *it;
+        }
+    } else {
+        UASSERT(v3Global.opt.cmake(), "at least cmake or gmake must be chosen in V3Options.cpp");
+        cmd << "cd " << v3Global.opt.makeDir() << " && ";
+        cmd << "cmake";
+        for (V3StringList::const_iterator it = makeFlags.begin(); it != makeFlags.end(); ++it) {
+            cmd << ' ' << *it;
+        }
+        cmd << ' ' << V3Os::getcwd() << " && ";
+        cmd << "cmake --build . ";
+        if (jobs == 0) {
+            cmd << " -j";
+        } else if (jobs > 1) {
+            cmd << " -j " << jobs;
+        }
+    }
+    const std::string cmdStr = cmd.str();
+
+    const int exit_code = V3Os::system(cmdStr);
+    if (exit_code != 0) {
+        v3error(cmdStr << " exitted with " << exit_code << std::endl);
+        exit(exit_code);
+    }
+}
+
+//######################################################################
+
+int main(int argc, char** argv, char** env) {
+    // General initialization
+    std::ios::sync_with_stdio();
+
+    time_t randseed;
+    time(&randseed);
+    srand(static_cast<int>(randseed));
+
+    // Post-constructor initialization of netlists
+    v3Global.boot();
+
+    // Preprocessor
+    // Before command parsing so we can handle -Ds on command line.
+    V3PreShell::boot(env);
+
+    // Command option parsing
+    v3Global.opt.bin(argv[0]);
+    string argString = V3Options::argString(argc-1, argv+1);
+    v3Global.opt.parseOpts(new FileLine(FileLine::commandLineFilename()),
+                           argc-1, argv+1);
+
+    // Validate settings (aka Boost.Program_options)
+    v3Global.opt.notify();
+
+    V3Error::abortIfErrors();
+
+    // Can we skip doing everything if times are ok?
+    V3File::addSrcDepend(v3Global.opt.bin());
+    if (v3Global.opt.skipIdentical().isTrue()
+        && V3File::checkTimes(v3Global.opt.makeDir() + "/" + v3Global.opt.prefix()
+                              + "__verFiles.dat", argString)) {
+        UINFO(1,"--skip-identical: No change to any source files, exiting\n");
+        exit(0);
+    }
+    // Undocumented debugging - cannot be a switch as then command line
+    // would mismatch forcing non-identicalness when we set it
+    if (!V3Os::getenvStr("VERILATOR_DEBUG_SKIP_IDENTICAL", "").empty()) {
+        v3fatalSrc("VERILATOR_DEBUG_SKIP_IDENTICAL w/ --skip-identical: Changes found\n");
+    }
+
+    if (v3Global.opt.verilate()) {
+        execVerilate(argString);
+    } else {
+        UINFO(1, "Option --no-verilate: Skip Verilation\n");
+    }
+
+    if (v3Global.opt.build()) {
+        execBuildJob();
+    }
```

Comment:
```suggestion
    if (v3Global.opt.build()) execBuildJob();
```

(As clang-format will reformat it to one line anyways)